src/corona_hakab_model/consts.py

```diff
@@ -273,27 +318,100 @@ class ImmuneTerminalState(ImmuneState, TerminalState):
+            probability=...
+        )
 
-        silent.add_transfer(
-            asymptomatic, self.silent_to_asymptomatic_days, self.silent_to_asymptomatic_probability,
+        pre_symptomatic.add_transfer(
+            mild_condition,
+            duration=self.pre_symptomatic_to_mild_condition_days,
+            probability=...
         )
-        silent.add_transfer(symptomatic, self.silent_to_symptomatic_days, ...)
 
-        symptomatic.add_transfer(
-            asymptomatic, self.symptomatic_to_asymptomatic_days, self.symptomatic_to_asymptomatic_probability,
+        mild_condition.add_transfer(
+            need_close_medical_care,
+            duration=self.mild_to_close_medical_care_days,
+            probability=0.2375,
+        )
+        mild_condition.add_transfer(
+            need_icu,
+            duration=self.mild_to_need_icu_days,
+            probability=0.0324
+        )
+        mild_condition.add_transfer(
+            pre_recovered,
+            duration=self.mild_to_pre_recovered_days,
+            probability=...
         )
-        symptomatic.add_transfer(hospitalized, self.symptomatic_to_hospitalized_days, ...)
 
-        hospitalized.add_transfer(icu, self.hospitalized_to_icu_days, self.hospitalized_to_icu_probability)
-        hospitalized.add_transfer(asymptomatic, self.hospitalized_to_asymptomatic_days, ...)
+        need_close_medical_care.add_transfer(
+            need_icu,
+            duration=self.close_medical_care_to_icu_days,
+            probability=0.26
+        )
+        need_close_medical_care.add_transfer(
+            mild_condition,
+            duration=self.close_medical_care_to_mild_days,
+            probability=...
+        )
 
-        icu.add_transfer(
-            hospitalized, self.icu_to_hospitalized_days, self.icu_to_hospitalized_probability,
+        need_icu.add_transfer(
+            deceased,
+            self.need_icu_to_deceased_days,
+            probability=0.0227
+        )
+        need_icu.add_transfer(
+            improving_health,
+            self.need_icu_to_improving_days,
+            probability=...
         )
-        icu.add_transfer(deceased, self.icu_to_deceased_days, ...)
 
-        asymptomatic.add_transfer(recovered, self.asymptomatic_to_recovered_days, ...)
+        improving_health.add_transfer(
+            need_icu,
+            duration=self.improving_to_need_icu_days,
+            probability=0.22
+        )
+        improving_health.add_transfer(
+            pre_recovered,
+            duration=self.improving_to_pre_recovered_days,
+            probability=0.39
+        )
+        improving_health.add_transfer(
+            mild_condition,
+            duration=self.improving_to_mild_condition_days,
+            probability=...
+        )
+
+        pre_recovered.add_transfer(
+            recovered,
+            duration=self.pre_recovered_to_recovered_days,
+            probability=...
+        )
+
+        asymptomatic.add_transfer(
```

Comment:
Asymp should go directly to recovered, not to pre-recovered.

Reply:
Fixed